messaging/src/main/java/org/axonframework/eventhandling/pooled/PooledStreamingEventProcessor.java

```diff
@@ -167,7 +168,7 @@ protected PooledStreamingEventProcessor(PooledStreamingEventProcessor.Builder bu
      *     <li>The {@link ErrorHandler} is defaulted to a {@link PropagatingErrorHandler}.</li>
      *     <li>The {@link MessageMonitor} defaults to a {@link NoOpMessageMonitor}.</li>
      *     <li>The {@code initialSegmentCount} defaults to {@code 16}.</li>
-     *     <li>The {@code initialToken} function defaults to {@link StreamableMessageSource#createTailToken()}.</li>
+     *     <li>The {@code initialToken} function defaults to {@link StreamableMessageSource#createHeadToken()}.</li>
```

Comment:
This is a bit to short. Now, people can read this as if the PSEP will skip all previous events for an initial token. Which isn't true, as it will make a `ReplayToken` for which the `tokenAtReset` is the head token.

Reply:
done :)